client/src/components/RankingQueryResultList.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import RankingQueryResultItem from './RankingQueryResultItem';
+
+function RankingQueryResultList({ query, queryValue }) {
+  const actualQueryValue = queryValue;

```

Comment:
nem biztos hogy át kell nevezni ez a queryValue-t ha csak annyit csinálunk vele hogy továbbadjuk a RankingQueryResultItem-nek. Azonos néven is továbbadható nem?

Reply:
Vagy itt volt az eslint-es errorod?

Reply:
Ezt javítottam!